.gitignore

```diff
@@ -1,5 +1,5 @@
 target/*
-.idea/*
```

Comment:
needed to avoid .idea folders inside code samples directory ( when opening a code sample in intellij it is created)